src/core/xfa/template.js

```diff
@@ -1833,9 +1835,20 @@ class Draw extends XFAObject {
       children: [],
     };
 
-    const assist = this.assist ? this.assist[$toHTML]() : null;
+    const assist = this.assist;
     if (assist) {
-      html.attributes.title = assist;
+      const assistHTML = this.assist[$toHTML]();
```

Comment:
I might be misunderstanding the code, but would the slightly more compact `const assistHTML = assist[$toHTML]();` work?